docs/.vitepress/config.js

```diff
@@ -350,42 +350,125 @@ function getDocumentDbSidebar() {
 function getEventStoreSidebar() {
   return [
     {
-      text: 'Schema Objects',
-      link: '/guide/events/schema'
+      text: 'Introduction',
+      link: '/guide/events/index'
     },
     {
-      text: 'Stream identity',
-      link: '/guide/events/identity'
+      text: 'Appending events',
```

Comment:
I think the outline looks good. I think I saw a note that you'd talk about features for dealing with concurrency in this slot, and I think that's probably fine.

Reply:
Yup, I'd like to add it there. If it grows too big, then we could nest it.